tenets/codelingo/cockroachdb/avoid-bool-params/example.go

```diff
@@ -0,0 +1,22 @@
+//Package main is an example package
+package main
+
+func main() {
+	doThisOrThat(false)
+}
+
+func doThisOrThat(flag bool) { // Issue
```

Comment:
add tests for 

```
func someFunc(a string, b bool) { ... }
```

```
func someFunc(b bool, a string) { ... }
```

```
func someFunc(a string, b bool, c string) { ... }
```

```
func someFunc(b bool, c string, d bool) { ... }
```

Reply:
Removed. Those test cases and updated expected file have been changed in source tenet